### PR TITLE
feat: export improvements 

### PR DESCRIPTION
**Pull request by RobCatton**
# Description

-   To improve multiple areas of the exports page.

# Testing instructions

-   Run locally and manually pass in data or add sleeps to see features.

# Type of change

-   [x] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

### CODE REVIEW

File: repos/fdbt-site/src/design/modules/_components.scss [hunk 1]
```diff
@@ -135,3 +135,7 @@
         color: govuk-shade(govuk-colour('red'), 20%);
     }
 }
+
+.remove-top-margin {
```
Comment (jacsamell) [inline repos/fdbt-site/src/design/modules/_components.scss:139]: There is a gov-uk margin 0 style already

File: repos/fdbt-site/src/design/modules/_spinner.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
+
+.lds-spinner {
```
Comment (jacsamell) [inline repos/fdbt-site/src/design/modules/_spinner.scss:2]: What does lds mean?

File: repos/fdbt-site/src/design/modules/_spinner.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    color: white;
+    display: inline-block;
+    position: relative;
+    width: 80px;
+    height: 80px;
+}
+.lds-spinner div {
```
Comment (jacsamell) [inline repos/fdbt-site/src/design/modules/_spinner.scss:9]: Is it worth refactoring this and putting it inside the top level class above? What do you think @JwanKhalaf ?

File: repos/fdbt-site/src/pages/products/exports.tsx [hunk 4]
```diff
@@ -60,39 +81,59 @@ const Exports = ({ csrf, myFaresEnabled, exportEnabled, operatorHasProducts }: G
...
+                                        </tr>
+                                    ))}
+                                </tbody>
+                            </table>
+                        )}
+
+                        {showPopup && (
+                            <InfoPopup
+                                title="We are preparing your export"
+                                text="Your export will take a few seconds to show in the table below."
```
Comment (jacsamell) [inline repos/fdbt-site/src/pages/products/exports.tsx:133]: Minor: is the table below or is it hidden at this point?

File: repos/fdbt-site/src/design/modules/_components.scss (hunk 1 above)
Comment (RobCatton) [reply repos/fdbt-site/src/design/modules/_components.scss:139]: Ah yeah my bad.

File: repos/fdbt-site/src/design/modules/_spinner.scss (hunk 2 above)
Comment (RobCatton) [reply repos/fdbt-site/src/design/modules/_spinner.scss:2]: no idea! will remove

File: repos/fdbt-site/src/pages/products/exports.tsx (hunk 4 above)
Comment (RobCatton) [reply repos/fdbt-site/src/pages/products/exports.tsx:133]: Yep - checked this with Dominic.

File: repos/fdbt-site/src/design/modules/_spinner.scss (hunk 3 above)
Comment (RobCatton) [reply repos/fdbt-site/src/design/modules/_spinner.scss:9]: Done!

**Review by jacsamell — APPROVED** [review]

**Pull request merged**